schedule/yam/agama_auto_iscsi.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: agama_auto_iscsi
+description: >
+  Prepare url for agama.auto boot parameter, boot, perform auto-installation with agama on iscsi disk.
```

Comment:
no need to explain how it works, just what does from functiona point of view, for example, next build will be `inst.auto` so then the code requires maintenance. Use `unattended` instead of `auto-installation`, 'Agama' instead of 'agama', should be `iscsi_ibft` and description should have that too.

Reply:
thanks for review, I updated it.